packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -238,16 +248,17 @@ class TextSpan {
 /// changes, return to step 2. If the text to be painted changes,
 /// return to step 1.
 class TextPainter {
+  /// Creates a text painter that paints the given text.
```

Comment:
say what happens if you omit the text (namely that the object is created but that we will crash if you try to paint() before setting the text to a non-null value? I think? We should probably be more graceful about that...)